lib/carrierwave/sequel.rb

```diff
@@ -11,6 +11,43 @@ def mount_uploader(column, uploader, options={}, &block)
       raise "You need to use Sequel 3.0 or higher. Please upgrade." unless ::Sequel::Model.respond_to?(:plugin)
       super
 
+      mod = Module.new
+      include mod
+      mod.class_eval <<-RUBY, __FILE__, __LINE__+1
+        def #{column}=(new_file)
+          if !(new_file.blank? && send(:#{column}).blank?)
+            modified!
+          end
+          super
+        end
+
+        def remove_#{column}!
```

Comment:
I see all of that but I thought that [ActiveRecord's version](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/orm/activerecord.rb#L80) shouldn't call `write_#{column}_identifier` (and even change column) since we trust [this line of documentation](https://github.com/carrierwaveuploader/carrierwave/blob/master/lib/carrierwave/mount.rb#L89). But now I see that I'm incorrect: [this test](https://github.com/carrierwaveuploader/carrierwave/blob/master/spec/uploader/remove_spec.rb#L50) is a proof. Sorry.